docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -24,6 +24,30 @@ This list is a work in progress.
 Expect updates as new breaking changes are introduced.
 :::
 
+### Hidden and PigmentHidden components removed
+
+The deprecated `Hidden` and `PigmentHidden` components have been removed.
+
+Use the `sx` prop to replace `implementation="css"`:
+
+```diff
+-<Hidden implementation="css" xlUp><Paper /></Hidden>
++<Paper sx={{ display: { xl: 'none', xs: 'block' } }} />
+```
+
+```diff
+-<Hidden implementation="css" mdDown><Paper /></Hidden>
++<Paper sx={{ display: { xs: 'none', md: 'block' } }} />
+```
```

Comment:
@brijeshb42 this won't work when replacing `PigmentHidden`, right?

How should users migrate? `useMediaQuery`?